src/types/hex-color-code.ts

```diff
@@ -1,9 +1,10 @@
-import {STRules} from '../rules';
 import {StrongType, makeStrong} from '../strong-type';
 
+import {STRules} from '../rules';
+
 export type StrongHexColorCode = StrongType<string>;
 
-export function makeHexColorCode(fallback: string, initial?: string | null): StrongHexColorCode {
+export function makeHexColorCode(fallback: string, initial: string | null | undefined): StrongHexColorCode {
```

Comment:
- [ ] Change `| undefined` type on `initial` to use the optional operator instead `?`.